integtest/spec/helper/dest.rb

```diff
@@ -235,6 +235,11 @@ def skip_link_check
       self
     end
 
+    def warn_link_check
```

Comment:
We can use this elsewhere in integration tests, but as of right now this isn't doing anything.

Reply:
Good point, I've removed it in 24d9d57f552c65e47f9493f7557962a7a6b4baa7